src/services/Utils.ts

```diff
@@ -18,10 +18,10 @@ import rundef from 'rundef';
 
 export class Utils {
     static range = (min: number, max: number): number[] => {
-        const len = max - min - 1;
+        const len = max - min + 1;
         const arr = new Array(len);
-        for (let i = 0; i <= len; i++) {
-            arr[i] = min + 1 + i;
+        for (let i = 0; i < len; i++) {
+            arr[i] = min + i;
         }
```

Comment:
here i runs from 0 to len - 1 so covering each array element. this could have gone out of bounds previously, but since it's a dynamically typed language it didn't show. finally computing the value is just offsetting i by min, no need to add a +1 there.